Branch-SDK/Branch-SDK/BNCEncodingUtils.m

```diff
@@ -199,7 +199,7 @@ + (NSString *)sanitizedStringFromString:(NSString *)dirtyString {
 
 + (NSData *)encodeDictionaryToJsonData:(NSDictionary *)dictionary {
     NSString *jsonString = [BNCEncodingUtils encodeDictionaryToJsonString:dictionary];
-    return [NSData dataWithBytes:[jsonString UTF8String] length:jsonString.length];
+    return [NSData dataWithBytes:jsonString.UTF8String length:strlen(jsonString.UTF8String)];
```

Comment:
OK, I'll do that. But in fact, `UTF8String` is a `property` of `NSString` pointing to a structure inside the string object, and there's no overhead calling twice.